pkg/pluginutil/pluginutil.go

```diff
@@ -92,6 +92,9 @@ func Logger() *logrus.Logger {
 		Hooks:     make(logrus.LevelHooks),
 	}
 	verbosity, _ := os.LookupEnv("DCOS_VERBOSITY")
+	if verbosity != "0" {
+		os.Setenv("DCOS_DEBUG", "1")
```

Comment:
Shall we keep this?

Reply:
Hm. If this is not set then we don't get any log messages from the client-go package. But I am not sure if we set it to zero by default, because otherwise it will always show the log messages right?

Reply:
We should only set it when `DCOS_VERBOSITY=2` (`-vv` was passed), and in the future we should improve client-go to make logging configurable through a config option instead of having to export an env var.